doc/api/http.md

```diff
@@ -1973,54 +1973,31 @@ Accept: text/plain\r\n
 \r\n
 ```
 
-Then `request.url` will be:
+To parse the url into its parts The WHATWG URL API can be used:
 
-<!-- eslint-disable semi -->
 ```js
-'/status?name=ryan'
+new URL(request.url, `http://${request.headers.host}`);
 ```
 
-To parse the url into its parts `require('url').parse(request.url)`
-can be used:
+When `request.url` is `'/status?name=ryan'` and `request.headers.host` is `'localhost:3000'`:
```

Comment:
This needs to be word-wrapped at 80 characters.

Reply:
I forgot it. Thank you for the review 😄